package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@material-ui/core": "4.11.0",
     "@material-ui/icons": "4.9.1",
+    "@types/react-router-dom": "^5.1.7",
```

Comment:
@jakeisnt from my understanding we should avoid having unpinned dependencies?

Reply:
yeah, it's not a huge deal with some packages, but as i mentioned before i noticed that the tailwindcss styles changed at some point last semester because the dependency wasn't fixed to a specific version, and having the package version change on `yarn install` could result in similar issues in the future

Reply:
even though it is not a big deal this should still be pinned

Reply:
just pinned the dependency